tests/test_cluster_inference.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+Testing cluster inference functions
+"""
+import numpy as np
+import pyrft as pr
+
+def test_find_clusters():
+    """ Testing the find_clusters function """
+    data = np.array([[1,0,1],[1,1,0]])

```

Comment:
pep8...

Reply:
In what way does this violate pep8? It's not showing up on my code checker.

Reply:
Happy to change it though

Reply:
OK, may be I need some updates. Would write it:
```suggestion
    data = np.array([[1, 0, 1], [1, 1, 0]])

```

Reply:
Happy to change this if this is pep8 (and I have done so), but these spacing things do not show up in my linter.

Reply:
I guess maybe there is an issue with my linter - I'm using the one built into spyder which seemed pretty good otherwise.